week-2/implement/tally.js

```diff
@@ -0,0 +1,23 @@
+function tally(arr) {
+    const count = {};
+  
+    for (let i = 0; i < arr.length; i++) {
+      let ele = arr[i];
+      if (count[ele]) {
```

Comment:
In line 6 this is not the correct way of checking if an item exists in the object. Instead, try using the `hasOwnProperty` method. Search how to use this methond and apply it here please!

Reply:
Otherwise nice work!